src/oc_reporter.erl

```diff
@@ -123,7 +128,14 @@ send_spans(Reporter, Config) ->
             ok;
         Spans ->
             ets:delete_all_objects(Buffer),
```

Comment:
should not be part of the catch? it can panic if the buffer table is lost.

Reply:
So would `ets:tab2list(Buffer)`. My thinking was that those should be crashes, while here we only want to catch exceptions from the reporter callback itself in order to not allow bad code in a third party reporter to bring it down.